packages/trader/src/AppV2/Hooks/useGetSymbolSearchResults.ts

```diff
@@ -0,0 +1,18 @@
+import React, { useMemo } from 'react';
+import useActiveSymbols from 'AppV2/Hooks/useActiveSymbols';
+import sortSymbols from 'AppV2/Utils/sort-symbols-utils';
+
+export const useGetSymbolSearchResults = (searchValue: string) => {
+    const { activeSymbols } = useActiveSymbols({});
+
+    const searchResults = useMemo(() => {
+        if (searchValue.trim() === '') return [];
+        return activeSymbols.filter(symbol => symbol.display_name.toLowerCase().includes(searchValue.toLowerCase()));
```

Comment:
`return searchValue.trim() ? activeSymbols.filter(symbol => symbol.display_name.toLowerCase().includes(searchValue.toLowerCase())) : []`

Reply:
Please, ignore if irrelevant

Reply:
either way is fine but i like the already existing pattern better.